maps/away/ships/nka/nka_merchant/nka_merchant.dm

```diff
@@ -23,6 +23,13 @@
 	max_speed = 1/(2 SECONDS)
 	fore_dir = NORTH
 	vessel_size = SHIP_SIZE_SMALL
+	scanimage = "nka_freighter.png"
+	designer = "New Kingdom of Adhomai"
```

Comment:
Might want NanoTrasen as designer here, too, considering they designed the base crafts?